pkg/pulumiyaml/ast/template.go

```diff
@@ -534,12 +596,6 @@ func TemplateSyntax(node *syntax.ObjectNode, description *StringExpr, configurat
 	}
 }
 
-func Template(description *StringExpr, configuration ConfigMapDecl, variables VariablesMapDecl, resources ResourcesMapDecl,
```

Comment:
This function is unused, and `pkg` is private so no need to worry about backwards compatibility.  Removed it because it clashes with the nice `Template` interface name.